pkr_extractor/walk.c

```diff
@@ -55,7 +55,7 @@ bool LoadDirectories(PKRDir *pkrDirs){
 	}
 
 	for(int i = 0; i<dirsRead; i++)
-		printf("%s has %d files\n", &pkrDirs[i].name, pkrDirs[i].numFiles);
+		printf("%s has %d files\n", pkrDirs[i].name, pkrDirs[i].numFiles);
```

Comment:
nit: `strncat` in other places implies that this should probably use `%.32s`. Other instances, too. I don't plan to fix it in this PR.